update-version.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -ex
+# read the value in version.txt increament the value if jenkins plugins needs to be updated
+
+version=$(<plugins/version.txt)
```

Comment:
what happens if the file is not there already  which is the case for all current users.